### DISCUSSION TITLE
I want you to implement a seeker preview Implement a seeker preview like MPC-HC I want to set the desired preview image magnification.

### DISCUSSION THREAD

**Discussion by fanz010** [mifi/lossless-cut#1870] [Ideas]
I want you to implement a seeker preview

Implement a seeker preview like MPC-HC

I want to set the desired preview image magnification.
There's a thumbnail function, but apart from that,
I think it would be useful if this software has a seek preview function.

MPC-HC is a well-known video playback software

To give you a better understanding of how it works, we have created a
video in MP4 format.
https://streamable.com/1r9cel
l
Thumbnails are useful to some extent, but
Please consider implementing it!

**Comment by fanz010** [comment]
To give you a better understanding of how it works, we have created a
video in MP4 format.
https://streamable.com/1r9cel

**Reply by mifi** [reply]
See "Thumbnails" under #483
